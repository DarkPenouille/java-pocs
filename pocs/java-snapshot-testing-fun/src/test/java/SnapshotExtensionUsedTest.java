import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.extension.ExtendWith;

import au.com.origin.snapshots.SnapshotMatcher;
import au.com.origin.snapshots.junit5.SnapshotExtension;

@ExtendWith(SnapshotExtension.class)
public class SnapshotExtensionUsedTest {

    @Test
    public void shouldUseExtension() {
        // Verify your snapshot
        SnapshotMatcher.expect("Hello Wolrd").toMatchSnapshot();
    }

    @Test
    public void exampleSnapshot() {
        SnapshotMatcher.expect("Hello Wolrd Again").toMatchSnapshot();
    }
}