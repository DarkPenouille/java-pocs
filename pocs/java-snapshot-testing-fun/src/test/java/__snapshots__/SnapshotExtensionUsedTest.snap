SnapshotExtensionUsedTest.exampleSnapshot=[
  "Hello Wolrd Again"
]


SnapshotExtensionUsedTest.shouldUseExtension=[
  "Hello Wolrd"
]